src/tus.js

```diff
@@ -58,7 +59,10 @@ async function sendTusRequest({
       uploadProgresses[label] = bytesUploaded
 
       // get all uploaded bytes for all files
-      const uploadedBytes = sumBy(streamLabels, (l) => uploadProgresses[l])
+      let uploadedBytes = 0
+      for (const l of streamLabels) {
+        uploadedBytes += uploadProgresses[l]
+      }
```

Comment:
this will probably err in typescript when strict null checks is enabled.  maybe better to do a `streamLables.reduce(...)`

Reply:
Why would this result in a TypeScript error? Reduce would definitely not help with this.

I strongly believe `array.reduce` [should never be used](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-reduce.md).

Reply:
the following code results in `Object is possibly 'undefined'.` typescript error when strict null checks is enabled (which i think makes typescript more accurate):

```js
const progresses: Record<string, number> = { a: 1, b: 2, c:3 }
const labels = ['a', 'b', 'c']
let sum = 0;
for (const l of labels) {
  sum += progresses[l]
}
```

but then this also gives the same error, so ignore what i just suggested 🤦‍♂️
```js
const sum = labels.reduce((acc, l) => acc + progresses[l], 0)
```

I misread your code and instead read it as a simple sum of elements. in that case this code does not produce the same ts error:
```
const sum = numbers.reduce((acc, n) => acc + n, 0)
```

but again ignore what i said.

as for using `array.reduce` i agree that it should be used sparingly, but as mentioned in your link it's nice for summing numbers:
> It's only somewhat useful in the rare case of summing up numbers, which is allowed by default.